ios/StripeSdk.swift

```diff
@@ -86,11 +86,14 @@ class StripeSdk: RCTEventEmitter, STPApplePayContextDelegate, STPBankSelectionVi
         }
 
         if  params["applePay"] as? Bool == true {
-            if let merchantIdentifier = self.merchantIdentifier, let merchantCountryCode = params["merchantCountryCode"] as? String {
-                configuration.applePay = .init(merchantId: merchantIdentifier,
-                                               merchantCountryCode: merchantCountryCode)
-            } else {
-                resolve(Errors.createError(ErrorType.Failed, "Either merchantIdentifier or merchantCountryCode is missing"))
+            do {
+                configuration.applePay = try ApplePayUtils.buildPaymentSheetApplePayConfig(
+                    merchantIdentifier: self.merchantIdentifier,
+                    merchantCountryCode: params["merchantCountryCode"] as? String ?? nil,
+                    paymentSummaryItems: params["paymentSummaryItems"] as? [[String : Any]] ?? nil
```

Comment:
do we need the `?? nil` in these two lines?

Reply:
nope, thanks for catching that! will remove